Classes/Private/Manager/Database/OCTRealmManager.m

```diff
@@ -21,7 +21,7 @@
 #undef LOG_LEVEL_DEF
 #define LOG_LEVEL_DEF LOG_LEVEL_VERBOSE
 
-static const uint64_t kCurrentSchemeVersion = 1;
+static const uint64_t kCurrentSchemeVersion = 7;
```

Comment:
I think we have different views on scheme version. What I was thinking about is to increment this number
- only for releases of objcTox
- only if Realm model has changed

So the main idea is to give "user" of objcTox (Antidote or Nao) easy way to upgrade on new version without loosing all data. On the other hand, while developing objcTox you don't want to increase number any time you change model. This should be only once per release.

So, the flow should be following. When you are changing model
- only if it is first change in release, add new version
- check if you should add something in migration block.

Please change version back to 1.